go.mod

```diff
@@ -5,15 +5,15 @@ go 1.16
 require (
 	cloud.google.com/go/asset v0.1.0
 	cloud.google.com/go/storage v1.10.0
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.19.0
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.11.0
-	github.com/Azure/azure-sdk-for-go/sdk/containerregistry/armcontainerregistry v0.2.0
-	github.com/Azure/azure-sdk-for-go/sdk/network/armnetwork v0.3.0
-	github.com/Azure/azure-sdk-for-go/sdk/postgresql/armpostgresql v0.2.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute v0.1.0
-	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/privatedns/armprivatedns v0.1.0
-	github.com/Azure/azure-sdk-for-go/sdk/resources/armresources v0.3.0
-	github.com/Azure/azure-sdk-for-go/sdk/storage/armstorage v0.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.20.0
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.12.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute v0.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerregistry/armcontainerregistry v0.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork v0.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/postgresql/armpostgresql v0.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/privatedns/armprivatedns v0.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources v0.2.0
```

Comment:
`armresources` is also downgraded

Reply:
Yes, the latest version for the new package is v0.2.0

See https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources